workers/ohsome_quality_analyst/base/indicator.py

```diff
@@ -42,7 +42,17 @@ class LayerDefinition:
 
 @dataclass
 class Result:
-    """The result of the Indicator."""
+    """The result of the Indicator.
+
+    Attributes:
+        timestamp_oqt (datetime): Timestamp of the creation of the indicator
+        timestamp_osm (datetime): Timestamp of the used OSM data
+            (e.g. Latest timestamp of the ohsome API results)
+        label (str): Traffic lights like quality label
+        value (float): The result value as float (0=>=1)
```

Comment:
To me `=>=` is very uncommon. Do you have a better representation like `0 ≥ value ≥ 1` or `[0, 1]`, …
```suggestion
        value (float): The result value as float ([0, 1])
```